cumulusci/tasks/bulkdata/snowfakery.py

```diff
@@ -676,3 +718,11 @@ def relevant_sobjects(self):
         with open(self.mapping_file, encoding="utf-8") as f:
             mapping = yaml.safe_load(f)
             return [m.get("sf_object") for m in mapping.values() if m.get("sf_object")]
+
+
+class RunningTotals:
+    errors: int = 0
+    successes: int = 0
+
+    def __repr__(self):  # pragma: no cover
```

Comment:
Please go ahead and cover code that is this easy to cover. I've seen too many mistakes in simple string formatting functions.